### ISSUE TITLE
problem with PyOpenSSL 16.1.0

### ISSUE THREAD

**Issue by codeforkjeff** [internetarchive/openlibrary-client#5]
After running `pip install openlibrary-client` I can't seem to import from olclient:

```
# python
Python 2.7.13 (default, Jan 19 2017, 14:48:08) 
[GCC 6.3.0 20170118] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from olclient.openlibrary import OpenLibrary
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jeff/.virtualenvs/env_aaaaarg/local/lib/python2.7/site-packages/olclient/__init__.py", line 17, in <module>
    from .openlibrary import OpenLibrary
  File "/home/jeff/.virtualenvs/env_aaaaarg/local/lib/python2.7/site-packages/olclient/openlibrary.py", line 13, in <module>
    import requests
  File "/home/jeff/.virtualenvs/env_aaaaarg/local/lib/python2.7/site-packages/requests/__init__.py", line 52, in <module>
    from .packages.urllib3.contrib import pyopenssl
  File "/home/jeff/.virtualenvs/env_aaaaarg/local/lib/python2.7/site-packages/requests/packages/urllib3/contrib/pyopenssl.py", line 54, in <module>
    import OpenSSL.SSL
  File "/home/jeff/.virtualenvs/env_aaaaarg/local/lib/python2.7/site-packages/OpenSSL/__init__.py", line 8, in <module>
    from OpenSSL import rand, crypto, SSL
  File "/home/jeff/.virtualenvs/env_aaaaarg/local/lib/python2.7/site-packages/OpenSSL/SSL.py", line 112, in <module>
    SSL_ST_INIT = _lib.SSL_ST_INIT
AttributeError: 'module' object has no attribute 'SSL_ST_INIT'
>>> 
```

Even running "pip list" produces this same error. I'm guessing pyOpenSSL 16.1.0 isn't playing nice with libssl 1.1.0e? I am running Debian testing.

If I clone the repository, change the version to 16.2.0 in `requirements.txt` and run `pip install .`, then openlibrary-client works perfectly.

Can this dependency be upgraded without causing problems for other users?

**Comment by mekarpeles** [comment]
@codeforkjeff -- confirmed. Yes, let's update the dependency to 16.2.0. Thank you for reporting. Will update here as the fix is merged in.

**Comment by mekarpeles** [comment]
Resolved by #6
